src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,107 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final double DEFAULT_LOAD_FACTOR = 0.75;
+
+    private int size = 0;
+    private int threshold = (int) (DEFAULT_INITIAL_CAPACITY * DEFAULT_LOAD_FACTOR);
+
+    private Node<K, V>[] table = (Node<K,V>[]) new Node[DEFAULT_INITIAL_CAPACITY];
+
+    static class Node<K, V> {
```

Comment:
```suggestion
    private static class Node<K, V> {
```
static clases should be after all methods